cmd/multiFaAcceleration/multiFaAcceleration_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 //Tests were made with the following cmds.
-//~/go/bin/multiFaAcceleration -windowSize 50 chr1 test.fa test.vel.expected.bed test.accel.expected.bed test.initialVel.expected.bed
+//~/go/bin/multiFaAcceleration -windowSize 50 -rawInitialVelBranchLengths test.RawInitial.bed -rawVelBranchLengths test.RawVel.bed chr1 test.fa test.vel.expected.bed test.accel.expected.bed test.initialVel.expected.bed
```

Comment:
How are we creating our tests for this program?  We should see if we can add more/different tests to cover the bug that made it through last time.